moveit_ros/planning_interface/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -124,7 +124,7 @@ class MoveItCpp
   MoveItCpp& operator=(MoveItCpp&& other);
 
   /** \brief Destructor */
-  ~MoveItCpp();
+  ~MoveItCpp() = default;
```

Comment:
How do these changes affect the test?

Reply:
It fixes the boost throwing bug

Reply:
I didn't check it again after updating the fake controller manager, I'm testing it right now locally, if there's no error I'll revert my changes